infinoted/infinoted-options.c

```diff
@@ -1003,6 +1013,7 @@ infinoted_options_free(InfinotedOptions* options)
   g_free(options->certificate_file);
   g_free(options->certificate_chain_file);
   g_free(options->root_directory);
+  inf_ip_address_free(options->listen_address);
```

Comment:
But this is not `free`, it is `inf_ip_address_free`, which does not like a `NULL` argument.